src/main/pg/pg.h

```diff
@@ -45,6 +45,7 @@ typedef void (pgResetFunc)(void * /* base */);
 
 typedef struct pgRegistry_s {
     pgn_t pgn;             // The parameter group number, the top 4 bits are reserved for version
+    uint8_t count;         // The number of elements in the group 
```

Comment:
Considering this is the number of elements in an array, would this not rather be `length` to match C terminology?

Reply:
done.